locales/fr-FR/App.json

```diff
@@ -6,9 +6,9 @@
   "Import wallet": "Importer un wallet",
   "Amount must be greater than {{ minAmountInAlph }}": "Le montant doit être supérieur à {{ minAmountInAlph }}",
   "Could not connect to the {{ currentNetwork }} network.": "Impossible de se connecter au réseau {{ currentNetwork }}.",
-  "Note that if activated, \"{{ name }}\" will not be the main address anymore.": "Notez que si l'option est activée, \"{{ name }}\" ne sera plus l'adresse principale.",
+  "Note that if activated, \"{{ name }}\" will not be the default address anymore.": "Notez que si l'option est activée, \"{{ name }}\" ne sera plus l'adresse principale.",
   "Error while sweeping address {{ from }}": "Erreur lors du nettoyage de l'address {{ from }}",
-  " Note that if activated, \"{{ address }}\" will not be the main address anymore.": " Notez que si l'option est activée, \"{{ address }}\" ne sera plus l'adresse principale.",
+  " Note that if activated, \"{{ address }}\" will not be the default address anymore.": " Notez que si l'option est activée, \"{{ address }}\" ne sera plus l'adresse principale.",
```

Comment:
@mvaivre I could use your help here, it's a simple change, but I don't wanna take my chances 😆

Reply:
Moreover, we should fix this so that we don't have these 2 entries, they are duplicated. We can do this cleanup after we merge the german translation, however, so that we don't confuse the contributors.

Reply:
" Notez que si l'option est activée, \"{{ address }}\" ne sera plus l'adresse par défaut."